src/main/java/de/retest/web/selenium/AutocheckingRecheckDriver.java

```diff
@@ -29,13 +29,10 @@ public AutocheckingRecheckDriver( final RemoteWebDriver wrapped ) {
 		options = RecheckOptions.builder().build();
 	}
 
-	// TODO Use RecheckWebOptions
-	public AutocheckingRecheckDriver( final RemoteWebDriver wrapped, final RecheckOptions options,
-			final AutocheckingCheckNamingStrategy namingStrategy ) {
+	public AutocheckingRecheckDriver( final RemoteWebDriver wrapped, final RecheckWebOptions options ) {
```

Comment:
1. Now that there is `RecheckWebOptions`, shouldn't we use it everywhere in recheck-web? For example, also in the `RecheckWebImpl` ctor?
2. Note that the changes of this PR are essentially breaking public API as one cannot pass `RecheckOptions` anymore. Therefore, only deprecate the old ones?
3. You could simplify this class as follows:

```java
public class AutocheckingRecheckDriver extends UnbreakableDriver {

	private RecheckWebImpl re;
	private final RecheckWebOptions options;
	private final AutocheckingCheckNamingStrategy namingStrategy;

	public AutocheckingRecheckDriver( final RemoteWebDriver wrapped ) {
		this( wrapped, RecheckWebOptions.builder().build() );
	}

	public AutocheckingRecheckDriver( final RemoteWebDriver wrapped, final RecheckWebOptions options ) {
		super( wrapped );
		this.options = options;
		namingStrategy = options.getNamingStrategy();
	}

	// ...

}
```

Reply:
Done